harness/tests/common/api/test_authentication.py

```diff
@@ -125,3 +127,24 @@ def test_auth_json_v1_upgrade() -> None:
         # Make sure we got exactly the master urls we expected.
         exp = {"http://firstmaster:80", "https://secondmaster:443", "http://thirdmaster:8080"}
         assert set(obj["masters"]) == exp, list(obj["masters"])
+
+
+def test_salt_and_hash() -> None:
```

Comment:
If this was to break I'd be so confused.

Could you please specify what the behavior is that you're testing and then test just for that?

Reply:
Thought process here was just "this is an easy and quick thing to test correctly, and anyone who touches code adjacent to this gets dinged on the lack of code coverage here, so I'll put a unit test here that exercises the actual function and checks its output so that future editors don't have to do this and don't end up putting a worse test here for the sake of coverage."

Reply:
:-) I think it's fine for a test -- I'm glad you added it.

The earlier comment was that I'd like some way to figure out if I was on call and this test broke _what_ the problem is likely to be. But it might not be easy to demonstrate. This is fine as-is.